server/src/test/java/org/elasticsearch/cluster/metadata/MetaDataIndexStateServiceTests.java

```diff
@@ -333,7 +333,7 @@ public void testAddIndexClosedBlocksReusesBlocks() {
     }
 
     public void testValidateShardLimit() {
-        int nodesInCluster = randomIntBetween(2,100);
+        int nodesInCluster = randomIntBetween(2,90);
```

Comment:
This change seems unrelated to types removal?

Reply:
Fixed by a separate merge in from master